Mathlib/Tactic/Linter/AdmitLinter.lean

```diff
@@ -0,0 +1,62 @@
+/-
+Copyright (c) 2024 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa, Adomas Baliuka
+-/
+import Lean.Elab.Command
+import Lean.Linter.Util
+
+/-!
+# The "admit" linter
+
+The "admit" linter flags usages of the `admit` tactic.
+
+The tactics `admit` and `sorry` are synonyms.
+The use of `sorry` is much more common and should be preferred.
+
+This linter is an incentive to discourage uses of `admit`, without being a ban.
+-/
+
+open Lean Elab
+
+namespace Mathlib.Linter.admit
+
+/-- The admit linter emits a warning on usages of `admit`. -/
+register_option linter.admit : Bool := {
```

Comment:
For the other linters, this option is usually put outside of the `linterName` namespace. Did you change this for a reason?

Reply:
I copied this from the refineLinter

Reply:
@adomani was there a reason this was done for the refineLinter? Should we change it there, too? Or just here?

Reply:
I don't have an opinion: I think that in copy-pasting something changed at one point and one of the two options became dominant!

Feel free to uniformize, if you want!

Reply:
I adjusted this PR to handle namespaces like what most other linters seem to do.